### DISCUSSION TITLE
Axum official site

### DISCUSSION THREAD

**Discussion by kei-ichi** [tokio-rs/axum#2547] [Ideas]
Just want to create an official **Axum** site included an Homepage and Document and everything else page like [Rocket](https://rocket.rs/) framework

Tech Stack:

Frontend:
- _Next.js_ or just pure _React.js_
- _Nuxt.js_ (if you guys prefer Vue) or just pure _Vue.js_

Backend:
- If using _Next.js_ -> _Express.js_
- Or just **Axum** (of course)
- If guys prefer something like blogs, change logs or something else -> _Strapi_ for _Headless CMS_

What do you guys think about this idea. (Feel free to roast me 👍🏼 )

**Comment by jplatte** [comment]
This has already been brought up a bunch of times, see discussion on https://github.com/tokio-rs/axum/issues/1811
